spec/helper.js

```diff
@@ -128,13 +128,16 @@ const reconfigureServer = changedConfiguration => {
       });
     }
     try {
+      var parseServer;
       const newConfiguration = Object.assign({}, defaultConfiguration, changedConfiguration, {
-        __indexBuildCompletionCallbackForTests: indexBuildPromise => indexBuildPromise.then(resolve, reject),
+        __indexBuildCompletionCallbackForTests: indexBuildPromise => indexBuildPromise.then(() => {
```

Comment:
I'm confused here, what is going on?  I don't see who wants this parse server back?  And I don't see how line 140 is or needs to be changed??  just trying to follow.

Reply:
We resolve with the parseServer instance now, that has access to everything, config. HttpServer etc...